api/graphql/schema.graphql

```diff
@@ -242,6 +247,7 @@ type Pool {
     @hasMany
     @canRoot(ability: "viewAssessmentPlan")
   poolSkills(type: PoolSkillType @where): [PoolSkill] @hasMany
+  roleAssignments: [RoleAssignment!] @hasManyThrough
```

Comment:
I feel like I should check the viewTeamMembers policy here but I'm not sure how to call that on the relevant team. Should I create a similar policy on Pool?